lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -205,11 +205,15 @@ int main(int argc, char* argv[]) {
   initialize_sentry();
 
   bool converged_access = false;
+  extern uint32_t session_max_rtx_count;
```

Comment:
Instead of defining a global variable, can we pass it into the relevant object/class?

Reply:
Yes, Removed the external variable and pass in a object.